src/extension.ts

```diff
@@ -83,20 +83,22 @@ export function activate(context: vscode.ExtensionContext): void {
 		})
 	);
 	context.subscriptions.push(
-		vscode.commands.registerCommand('xsoar.lint', (file: vscode.Uri | undefined) => {
+		vscode.commands.registerCommand('xsoar.preCommit', (file: vscode.Uri | undefined) => {
 			const fileToRun = getDirPath(file)
-			vscode.window.showQuickPick(['With tests', 'Without tests'], { placeHolder: 'Lint with tests?' }).then(option => {
-				if (option === 'With tests') {
-					dsdk.lint(fileToRun, true)
-				} else {
-					dsdk.lint(fileToRun, false)
+			vscode.window.showQuickPick(['Default configuration', 'Only tests', 'No docker'], { placeHolder: 'Select pre-commit configuration' }).then(option => {
```

Comment:
Maybe rename to `No docker based hooks` or `No docker hooks`?

Reply:
changed to `No docker based hooks`